index.html

```diff
@@ -77,6 +78,230 @@ <h3>LET'S CONNECT</h3>
         </li>      
        </ul>
     </section>
+    <section class="portfolio" id="portfolio">
+      <nav class="portfolio-border">
+        <img
+          class="portfolio-image"
+          alt="portfolio project"
+          src="Snapshoot_Portfolio1.png"
+        />
+        <div class="project-section">
+          <h3 class="project-name">Tonic</h3>
+          <div class="project-info">
+            <h4 class="project-client">CANOPY</h4>
+            <img src="Counter.png" class="counter" alt="counter" />
+            <span class="programming-language">Back End Dev</span>
+            <img src="Counter.png" class="counter" alt="counter" />
+            <span class="year">2015</span>
+          </div>
+
+          <p class="project-details">
+            A daily selection of privately personalized reads; no accounts or <br />
+            sign-ups required. 
+          </p>  
+          <ul class="languages">
+            <li class="dev-tool">html</li>
+            <li class="dev-tool">css</li>
+            <li class="dev-tool">javascript</li>
+          </ul>
+          <button class="btn" type="button">See Project</button>
+        </div>
+      </nav>
+
+      <nav class="portfolio-border">
```

Comment:
Kindly, by convention on your website, there should only be 1 NAV tag, I recommend changing it to a DIV